packages/errors/src/errors.ts

```diff
@@ -102,30 +102,42 @@ export const AllCypressErrors = {
 
         ${fmt.listItems(options)}`
   },
-  BROWSER_NOT_FOUND_BY_NAME: (browser: string, foundBrowsersStr: string[]) => {
+  BROWSER_NOT_SUPPORTED: (browser: string, foundBrowsersStr: string[]) => {
     let canarySuffix: PartialErr | null = null
 
     if (browser === 'canary') {
       canarySuffix = errPartial`\
           ${fmt.off('\n\n')}
-          Note: In ${fmt.cypressVersion(`4.0.0`)}, Canary must be launched as ${fmt.highlightSecondary(`chrome:canary`)}, not ${fmt.highlightSecondary(`canary`)}.
+          Note: Since ${fmt.cypressVersion(`4.0.0`)}, Canary must be launched as ${fmt.highlightSecondary(`chrome:canary`)}, not ${fmt.highlightSecondary(`canary`)}.
 
           See https://on.cypress.io/migration-guide for more information on breaking changes in 4.0.0.`
     }
 
     return errTemplate`\
-        Can't run because you've entered an invalid browser name.
+        Can't run because you've entered a browser that is not supported by Cypress.
 
-        Browser: ${fmt.highlight(browser)} was not found on your system or is not supported by Cypress.
+        Browser: ${fmt.highlight(browser)} is not supported by Cypress.
 
         Cypress supports the following browsers:
-        ${fmt.listItems(['electron', 'chrome', 'chromium', 'chrome:canary', 'edge', 'firefox'])}
+        ${fmt.listItems(['electron', 'chrome', 'chromium', 'chrome:canary', 'edge', 'firefox', 'webkit'])}
```

Comment:
Should we note here that webkit support is still experimental?

Reply:
@cacieprins @jennifer-shehane will incorporate these changes shortly